src/test/java/seedu/dictionote/commons/core/VersionTest.java

```diff
@@ -128,6 +133,15 @@ public void versionComparable_validVersion_equalIsCorrect() {
         assertTrue(one.equals(another));
     }
 
+    @Test
+    public void versionComparable_validVersion_equalIsNotCorrect() {
+        Version one;
+
+        one = new Version(0, 0, 0, false);
+        assertTrue(!(one.equals(null)));
+        assertTrue(!(one.equals("test")));
```

Comment:
Just a minor and optional thing, but you might want to consider using `assertFalse` here

Reply:
fixed